src/Modules/Meetings/Domain/MeetingGroupProposals/MeetingGroupProposalStatus.cs

```diff
@@ -1,15 +1,17 @@
-﻿using CompanyName.MyMeetings.BuildingBlocks.Domain;
+﻿using System.Reflection.Metadata;
+using CompanyName.MyMeetings.BuildingBlocks.Domain;
 
 namespace CompanyName.MyMeetings.Modules.Meetings.Domain.MeetingGroupProposals
 {
     public class MeetingGroupProposalStatus : ValueObject
     {
-        internal static MeetingGroupProposalStatus InVerification => new MeetingGroupProposalStatus("InVerification");
-        internal static MeetingGroupProposalStatus Accepted => new MeetingGroupProposalStatus("Accepted");
+        internal bool IsAccepted => Value == "Accepted";
+        internal static MeetingGroupProposalStatus CreateInVerification => new MeetingGroupProposalStatus("InVerification");
+        internal static MeetingGroupProposalStatus CreateAccepted => new MeetingGroupProposalStatus("Accepted");
```

Comment:
I added this prefix because this property behaves as a factory method and speaking for myself, I wouldn't expect that there is a new object creation happening. In fact I would add that "Create" prefix for every object creation, I just wanted to discuss it a little further by this sample PR. Maybe the best way would be to replace that property with factory method? I know it is not uncommon that the property returns a new object, but when the property return a new instance of the containing class, that sounds more like static factory rather than property, the difference is lack of braces IMHO.

Reply:
obviously VOs are immutable, we are aligned on that. My point was on the scenario where you compare VO by creating new VO in that process. They must be compared by their values of course, but I saw this is in the code also done by using that property(Accepted) returning new object, leading to a situation when a new object is created just to compare values.

Example: 
`public bool IsBroken() => _actualStatus == MeetingGroupProposalStatus.Accepted;`

So, in general, you agree that it would be reasonable to replace the above with:
`public bool IsBroken() => _actualStatus == MeetingGroupProposalStatus.IsAccepted;`

I think if there would be no logic regarding object creation, only constructor with string parameter would be sufficient. 
Yet the value of MeetingGroupProposalStatus is expected to be  "InVerification" and not "BeingVerified" or some other random, maybe it's not so much of a business logic, but I think it has some "tiny logic". It does protect itself from being created with invalid state.

I'll create a new PR without the renaming to not block it with this one discussion.